style.css

```diff
@@ -0,0 +1,568 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%;
+}
+
+body {
+  font-family: "Rubik", sans-serif;
+  line-height: 1;
+  font-weight: 400;
+  color: #555;
+}
+
```

Comment:
- [ ] While you were working on the desktop version, it affected the mobile version. Kindly make sure the mobile version remains as expected.
- [ ] For the desktop version breakpoint, you should use the media queries value for the `min-width` property in the range of `768px` or wider